egi-qc/ansible-umd
# Add testing with molecule and refactor code

Major changes:

  - Add molecule with default scenario
    - test coverage only includes CEntOS7
    - test coverage only includes default file existence
  - Refactor tasks:
    - Include OS-specific variables before OS-specific tasks
    - Change syntax of task inclusion from include: to include_tasks:
    - Change failure message condition to check if the release version
      exists for the OS
